src/Spectre.libWavelet/Convolution.h

```diff
@@ -39,6 +42,6 @@ class Convolution
     /// <param name="kernel">Kernel to be used.</param>
     /// <param name="signal">Signal to be convolved.</param>
     /// <returns>Filtered signal.</returns>
-    Signal Convolve(const Signal& kernel, const Signal& signal);
+    Signal Convolve(Kernel& kernel, const Signal& signal) const;
```

Comment:
... and place it here? (`const Kernel& kernel`)? I just thought that signature would be more explicit on IDE hints.